meilisearch-http/tests/common.rs

```diff
@@ -111,17 +111,19 @@ impl Server {
 
 
     pub async fn wait_update_id(&mut self, update_id: u64) {
-        loop {
+        // try 10 times to get status, or panic to not wait forever
+        for _ in 1..10 {
```

Comment:
This will not loop for 10 rounds but only 9 :)

Reply:
of course...